db/migrate/20220914131308_add_users_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
+  end
+end
```

Comment:
Same here, Please edit this migration because now it's creating a foreign key as user_id and It's supposed to be author_id like the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).